src/MessageHandler/ActivityPub/Inbox/ActivityHandler.php

```diff
@@ -63,10 +63,10 @@ public function __invoke(ActivityMessage $message): void
 
         try {
             if (isset($payload['actor']) || isset($payload['attributedTo'])) {
-                if (!$this->verifyInstanceDomain($payload['actor'] ?? $payload['attributedTo'])) {
+                if (!$this->verifyInstanceDomain($payload['actor'] ?? $this->manager->getActorFromAttributedTo($payload['attributedTo']))) {
                     return;
                 }
-                $user = $this->manager->findActorOrCreate($payload['actor'] ?? $payload['attributedTo']);
+                $user = $this->manager->findActorOrCreate($payload['actor'] ?? $this->manager->getActorFromAttributedTo($payload['attributedTo']));
```

Comment:
Keep in mind that `getActorFromAttributedTo` can still return null, possibly set `$user` to null here.

Reply:
since `findActorOrCreate` is fine with getting `null` as the `url` and can return `null` itself I think this is not a problem

Reply:
OK, noted. ^^